### ISSUE TITLE
SHT85

### ISSUE THREAD

**Issue by starodum** [ursl/tessie#29]
It was already reported that SHT85 has problems. But today we (me and  @jonamotta) observed the following. When we connect the LV PSs (ROHDE&SCHWARZ HMP4040)  to our cold box the data from the sensor became corrupted in a way that Tair=-45C and RH=-99 (and due point is NAN). Has anyone observed similar behavior and is there a solution of this problem.
I also must say that we did not yet make a proper grounding of every our device. Could this be a problem?

**Comment by ursl** [comment]
Yes. I would predict that your SHT85 is close to breaking and soon will not provide "real" RH readouts anymore. 

You could try the current HEAD (https://github.com/ursl/tessie/commit/4941eb357212b4120afa80fd907924288183615e) which provides a "throttling" of the N2 flow such that the RH stays above 5%. It might prolong the lifetime of your *next* SHT85.

**Comment by ursl** [comment]
Maybe I should add that I observe failing SHT85 readout (for those sensors that are about to break) also when turning on the valves.

**Comment by starodum** [comment]
Thank you, Urs. But strange thing is that as soon as we disconnect lv power supply cables Tair and RH values become correct again. We repeated this several times with the same result.

**Comment by ursl** [comment]
Hm. I have seen this behavior with the valves. I do not know our grounding scheme here. We can discuss this tomorrow in person in front of our boxes.

**Comment by ursl** [comment]
A note: We are considering to replace the SHT85 with the [HYT 223](https://www.ist-ag.com/en/products/humidity-module-integrated-heater), which allows in-situ re-conditioning (something like 2' after loading the modules)

**Comment by ursl** [comment]
closed because of lack of activity
